### PR TITLE
Removed unwanted font files.

### PR DESCRIPTION
**Pull request by kulothunganug**
### Description of Changes
* Removed unwanted font files because `Kivy` already have these fonts and registered those.

### CODE REVIEW

File: kivymd/font_definitions.py [hunk 1]
```diff
@@ -12,13 +12,6 @@
 from kivymd import fonts_path
 
 fonts = [
-    {
-        "name": "Roboto",
-        "fn_regular": fonts_path + "Roboto-Regular.ttf",
-        "fn_bold": fonts_path + "Roboto-Bold.ttf",
-        "fn_italic": fonts_path + "Roboto-Italic.ttf",
-        "fn_bolditalic": fonts_path + "Roboto-BoldItalic.ttf",
-    },
```
Comment (HeaTTheatR) [inline kivymd/font_definitions.py:21]: Actually, these fonts are used.

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/font_definitions.py (hunk 1 above)
Comment (kulothunganug) [reply kivymd/font_definitions.py:21]: these fonts are already exists in https://github.com/kivy/kivy/tree/master/kivy/data/fonts so we don't want to use separately.

Comment (HeaTTheatR) [reply kivymd/font_definitions.py:21]: Have you compared these fonts? They differ.

Comment (kulothunganug) [reply kivymd/font_definitions.py:21]: No, How do i compare them?

Comment (kulothunganug) [reply kivymd/font_definitions.py:21]: Mostly it would be same.

Comment (kulothunganug) [reply kivymd/font_definitions.py:21]: refs
https://github.com/kivy/kivy/blob/5e1311913722387242e1a3a48267333843a0e4a9/kivy/config.py#L867
https://github.com/kivy/kivy/blob/5e1311913722387242e1a3a48267333843a0e4a9/kivy/core/text/__init__.py#L94

**Comment by HeaTTheatR** [comment]
Kivy and KivyMD fonts are very different. We have no right to delete them.

**Pull request closed without merging**